doc/api/process.md

```diff
@@ -362,6 +362,14 @@ process.stdin.resume();
 process.on('SIGINT', () => {
   console.log('Received SIGINT.  Press Control-D to exit.');
 });
+
+// Using a single function to handle multiple signals
+process.on('SIGINT', handle);
+process.on('SIGTERM', handle);
+
+function handle(signal, code) {
```

Comment:
I think the documentation needs to mention that these new arguments are provided.

Reply:
I am sorry but why is the signal added as well? The handler itself already listens to the signal and that is redundant therefore?